air/src/lib.rs

```diff
@@ -339,6 +353,9 @@ impl miden_core::ToElements<Felt> for PublicInputs {
         let mut result = self.stack_inputs.to_vec();
         result.append(&mut self.stack_outputs.to_vec());
         result.append(&mut self.program_info.to_elements());
+        // TODO: Append transcript state to PI elements after recursion supports it.
+        // let cap: [Felt; 4] = self.precompile_transcript_state.into();
+        // result.extend_from_slice(&cap);
```

Comment:
Q: Should we probably just uncomment these and deactivate the recursive verifier test, Phil is also probably doing the same for his PR which changes the number of committed columns